mobile/src/main/java/org/openhab/habdroid/core/OnUpdateBroadcastReceiver.java

```diff
@@ -0,0 +1,49 @@
+package org.openhab.habdroid.core;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import org.openhab.habdroid.ui.OpenHABMainActivity;
+import org.openhab.habdroid.util.Constants;
+
+public class OnUpdateBroadcastReceiver extends BroadcastReceiver {
+    private static final String TAG = OpenHABMainActivity.class.getSimpleName();
+
+    private static final String UPDATE_LOCAL_CREDENTIALS = "1.8.0.8";
```

Comment:
Shouldn't be this the next version number? Will be 2.0 https://github.com/openhab/openhab.android/pull/341#issuecomment-322299485

Reply:
Fixed!